src/standalone/Vegetation/canopy_parameterizations.jl

```diff
@@ -42,19 +44,20 @@ BeerLambertModel, along with the PAR, LAI, extinction coefficient K, and solar
 zenith anlgle.
 """
 function plant_absorbed_ppfd(
-    RT::BeerLambertModel,
+    RT::BeerLambertModel{FT},
     PAR::FT,
     LAI::FT,
     K::FT,
-    θs::FT,
+    _,
+    _,
```

Comment:
I think just one extra arg

Reply:
The method for the BeerModel doesn't use the solar zenith angle (it is only used for the computation of `K` which is done outside the function), I actually added it into the function signature when I added the TwoStreamModel. Since it isn't used within the function anywhere should it be an extra arg here like this?

Reply:
oh, yes! I missed that theta_s was dropped. this makes sense, thank you.

just fyi - you can define methods in Julia with a different # of arguments, but here we always call the function with the same arguments (from canopy.jl). so I think we have to define them with the same # of arguments, otherwise I think we'd run into "No Method Matching" type errors...